### ISSUE TITLE
A tiny improvement of readability in a single line. examples/systemd/vnstat.service: line 9: --nodaemon instead of -n

### ISSUE THREAD

**Issue by e-Gyi-qO** [vergoh/vnstat#274]
For a tiny improvement of readability, substitute `-n` with `--nodaemon` at [examples/systemd/vnstat.service: line 9](https://github.com/vergoh/vnstat/blob/master/examples/systemd/vnstat.service#L9). That is,
```
ExecStart=/usr/sbin/vnstatd --nodaemon
```
instead of `ExecStart=/usr/sbin/vnstatd -n`

**Comment by vergoh** [comment]
Yes, looks like the same pattern repeats in several other files + also in the [vnstat-docker](https://github.com/vergoh/vnstat-docker) repository too. I'll get those sorted out as it doesn't hurt to improve readability.
